contracts/PBAB+Collabs/verse-flex/GenArt721CoreV2_VerseFlex.sol

```diff
@@ -168,6 +168,97 @@ contract GenArt721CoreV2_VerseFlex is ERC721, IGenArt721CoreV2_PBAB {
         randomizerContract = IRandomizer(_randomizerContract);
     }
 
+    /**
+     * @notice Mints a token from project `_projectId` and sets the
+     * token's owner to `_to`.
```

Comment:
Nit: can you please clarify here that this is using the provided hash to associate with the token (which may be centralized), rather than deriving this via on-chain mechanisms?

Reply:
Thanks for spotting @jakerockland! Fixed the comment.